bootstrap.php

```diff
@@ -154,539 +101,35 @@ function (AggregateType $aggregateType) use ($app) {
-            $eventManager
-        );
-
-        return $connection;
-    }
-);
-
-$app['dbal_event_store'] = $app->share(
-    function ($app) {
-        return $app['event_store_factory'](AggregateType::event());
-    }
-);
-
-$app['event_store'] = $app->share(
-    function ($app) {
-        return new \CultuurNet\UDB3\EventSourcing\CopyAwareEventStoreDecorator($app['dbal_event_store']);
-    }
-);
+$container->addServiceProvider(new CacheServiceProvider());
 
-$app['calendar_factory'] = $app->share(
-    function () {
-        return new CalendarFactory();
-    }
-);
+$container->addServiceProvider(new DatabaseServiceProvider());
 
-$app['cdbxml_contact_info_importer'] = $app->share(
-    function () {
-        return new CdbXmlContactInfoImporter();
-    }
-);
+$container->addServiceProvider(new EventServiceProvider());
 
 $container->addServiceProvider(new EventJSONLDServiceProvider());
 
-$app['event_calendar_repository'] = $app->share(
-    function ($app) {
-        return new \CultuurNet\UDB3\Event\ReadModel\Calendar\CacheCalendarRepository(
-            $app['event_calendar_cache']
-        );
-    }
-);
-
-$app['event_calendar_cache'] = $app->share(
-    function (Application $app) {
-        return $app['cache']('event_calendar');
-    }
-);
-
-$app['event_calendar_projector'] = $app->share(
-    function ($app) {
-        return new \CultuurNet\UDB3\Event\ReadModel\Calendar\EventCalendarProjector(
-            $app['event_calendar_repository']
-        );
-    }
-);
-
-$app['events_locator_event_stream_decorator'] = $app->share(
-    function (Application $app) {
-        return new OfferLocator($app['event_iri_generator']);
-    }
-);
-
-$app['event_repository'] = $app->share(
-    function ($app) {
-        $repository = new \CultuurNet\UDB3\Event\EventRepository(
-            $app['event_store'],
-            $app[EventBus::class],
-            [
-                $app['event_stream_metadata_enricher'],
-                $app['events_locator_event_stream_decorator']
-            ]
-        );
-
-        return $repository;
-    }
-);
-
-$app['logger_factory.resque_worker'] = $app::protect(
-    function ($queueName) use ($app) {
-        $redisConfig = [
-            'host' => '127.0.0.1',
-            'port' => 6379,
-        ];
-        if (extension_loaded('redis')) {
-            $redis = new Redis();
-            $redis->connect(
-                $redisConfig['host'],
-                $redisConfig['port']
-            );
-        } else {
-            $redis = new Predis\Client(
-                [
-                    'host' => $redisConfig['host'],
-                    'port' => $redisConfig['port']
-                ]
-            );
-            $redis->connect();
-        }
-        $socketIOHandler = new SocketIOEmitterHandler(new Emitter($redis), Logger::INFO);
-
-        return LoggerFactory::create($app->getLeagueContainer(), LoggerName::forResqueWorker($queueName), [$socketIOHandler]);
-    }
-);
-
 /** Production */
```

Comment:
This comment is no longer relevant and might best be removed.

Reply:
I re-ordered the whole file so it makes a bit more sense, this comment was indeed not necessary 👍